tools/Azure.Mcp.Tools.AzureManagedLustre/src/Commands/FileSystem/FileSystemImportJobCreateCommand.cs

```diff
@@ -0,0 +1,105 @@
+{
+    private const string CommandTitle = "Create AMLFS Import Job";
+
+    private readonly Option<string> _fileSystemOption = AzureManagedLustreOptionDefinitions.FileSystemOption;
+    private readonly Option<string[]> _importPrefixesOption = AzureManagedLustreOptionDefinitions.ImportPrefixesOption;
+    private readonly Option<string> _conflictResolutionModeOption = AzureManagedLustreOptionDefinitions.ConflictResolutionModeOption;
+    private readonly Option<int?> _maximumErrorsOption = AzureManagedLustreOptionDefinitions.MaximumErrorsOption;
+    private readonly Option<string> _adminStatusOption = AzureManagedLustreOptionDefinitions.AdminStatusOption;
+    private readonly Option<string> _jobNameOption = AzureManagedLustreOptionDefinitions.JobNameOption;
+
+    public override string Name => "create";
+
+    public override string Description =>
+        """
+        Creates a manual import job for an Azure Managed Lustre (AMLFS) file system. The import job scans the linked HSM/Blob container and imports specified path prefixes (or all when omitted) honoring the chosen conflict resolution mode. Use to hydrate the AMLFS namespace or refresh content.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new() { Destructive = false, ReadOnly = false };
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        RequireResourceGroup();
+        command.AddOption(_fileSystemOption);
+        command.AddOption(_importPrefixesOption);
+        command.AddOption(_conflictResolutionModeOption);
+        command.AddOption(_maximumErrorsOption);
+        command.AddOption(_adminStatusOption);
+        command.AddOption(_jobNameOption);
+    }
+
+    protected override FileSystemImportJobCreateOptions BindOptions(ParseResult parseResult)
+    {
+        var options = base.BindOptions(parseResult);
+        options.FileSystem = parseResult.GetValueForOption(_fileSystemOption);
+        var prefixes = parseResult.GetValueForOption(_importPrefixesOption);
+        options.ImportPrefixes = prefixes == null ? new List<string> { "/" } : prefixes.ToList();
+        options.ConflictResolutionMode = parseResult.GetValueForOption(_conflictResolutionModeOption) ?? "OverwriteAlways";
+        options.MaximumErrors = parseResult.GetValueForOption(_maximumErrorsOption) ?? -1;
+        options.AdminStatus = parseResult.GetValueForOption(_adminStatusOption) ?? "Active";
+        options.JobName = parseResult.GetValueForOption(_jobNameOption);
+        return options;
+    }
+
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var options = BindOptions(parseResult);
+        try
+        {
+            if (!Validate(parseResult.CommandResult, context.Response).IsValid)
+            {
+                return context.Response;
+            }
```

Comment:
There's an upcoming broad update where `Validate` should be called before `BindOptions`, mind updating that here.